Games/core/Game.js

```diff
@@ -791,12 +794,48 @@ module.exports = class Game {
         this.processAlertQueue();
         this.events.emit("meetingsMade");
 
+        this.vegMeeting = undefined;
+
         // Create next state timer
         this.createNextStateTimer(stateInfo);
     }
 
     createNextStateTimer(stateInfo) {
-        this.createTimer("main", stateInfo.length, () => this.gotoNextState());
+        this.createTimer("main", stateInfo.length, () => this.checkVeg());
+        this.checkAllMeetingsReady();
+    }
+
+    checkVeg() {
+        this.clearTimer("main");
+        this.clearTimer("secondary");
+        this.vegMeeting = this.createMeeting(VegReadyMeeting, "vegKickMeeting");
+        var vegMeetings = Object.values(this.history.states[this.currentState].meetings).filter(x => x.id !== this.vegMeeting.id);
+        for (let player of this.players) {
+            var addToKick = true;
+            if (player.alive) {
+                for (let i = 0; i < vegMeetings.length; i++) {
+                    if (vegMeetings[i].members[player.id] !== undefined) {
+                        if (vegMeetings[i].votes[player.id] === undefined) {
+                            if (vegMeetings[i].members[player.id].canVote && vegMeetings[i].members[player.id].canUpdateVote) {

```

Comment:
pardon me, why is there a need to check canupdatevote here?

because this functions is checking for people who have yet to vote for their meetings, what if they canVote and cannotUpdateVote (will this even occur)